vyper/types/convert.py

```diff
@@ -346,9 +344,9 @@ def to_bytes32(expr, args, kwargs, context):
     if input_type == 'bytes':
         if _len > 32:
             raise TypeMismatchException((
```

Comment:
Consistency, combined with below
```suggestion
            raise TypeMismatchException(
```